fs.go

```diff
@@ -36,11 +36,11 @@ func (ca *Cagent) FSWatcher() *FSWatcher {
 	}
 
 	for _, t := range ca.Config.FSTypeInclude {
-		ca.fsWatcher.AllowedTypes[strings.ToLower(t)] = struct{}{}
+		ca.fsWatcher.AllowedTypes[t] = struct{}{}
```

Comment:
This part was from earlier code I think, and looked correct. The filesystem types should be expressed in lower case (ext3, not Ext3 or EXT3)

Reply:
Oops. To much cut